configure.ac

```diff
@@ -162,6 +162,19 @@ then
 fi
 
 
+# Linux kernel TPM device Support
+AC_ARG_ENABLE([devtpm],
+    [AS_HELP_STRING([--enable-devtpm],[Enable use of TPM through the Linux kernel driver (default: disabled)])],
+    [ ENABLED_DEVTPM=$enableval ],
+    [ ENABLED_DEVTPM=no ]
+    )
+
+if test "x$ENABLED_DEVTPM" = "xyes"
+then
+    AM_CFLAGS="$AM_CFLAGS -DWOLFTPM_LINUX_DEV"
+fi
+
```

Comment:
Please add a conditional below and only include `tpm2_linux.c` when `BUILD_DEVTPM` is set by adding this check in `src/include.am`.

```
AM_CONDITIONAL([BUILD_DEVTPM], [test "x$ENABLED_DEVTPM" = "xyes"])
```

Also add a `# output config summary` entry below after line 361.

Reply:
Output config summary and conditional build check added. Both of these changes go to configure.am, I guess the "src/include.am" mentioning was a typo.